src/index.js

```diff
@@ -0,0 +1,151 @@
+const state = {
+	cityLocation: { lat: 28.5421109, lon: -81.3790304 },
+};
+
+const loadControls = () => {
+	state.tempValueEl = document.getElementById('tempValue');
+	state.increaseTempControl = document.getElementById('increaseTempControl');
+	state.decreaseTempControl = document.getElementById('decreaseTempControl');
+	state.tempValue = parseInt(state.tempValueEl.textContent);
+	state.gardenLandscape = document.getElementById('landscape');
+	state.gardenLandscapeValue = state.gardenLandscape.textContent;
+	state.cityLocationEl = document.getElementById('cityNameInput');
+	state.skyDisplay = document.getElementById('sky');
+	state.skyDisplayValue = state.skyDisplay.textContent;
+};
+
+const increaseTemp = () => {
+	state.tempValue += 1;
+	state.tempValueEl.textContent = `${state.tempValue}`;
+	changeColorAndLandscape();
+};
+
+const decreaseTemp = () => {
+	state.tempValue -= 1;
+	state.tempValueEl.textContent = `${state.tempValue}`;
+	changeColorAndLandscape();
+};
+
+const changeColorAndLandscape = () => {
+	let temp = state.tempValue;
+	if (temp >= 80) {
+		state.tempValueEl.classList = 'red';
+		state.gardenLandscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+	} else if (temp < 80 && temp >= 70) {
+		state.tempValueEl.classList = 'orange';
+		state.gardenLandscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+	} else if (temp < 70 && temp >= 60) {
+		state.tempValueEl.classList = 'yellow';
+		state.gardenLandscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+	} else if (temp < 60 && temp >= 50) {
+		state.tempValueEl.classList = 'green';
+		state.gardenLandscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+	} else {
+		state.tempValueEl.classList = 'teal';
+		state.gardenLandscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+	}
+};
+
+const convertKToF = (k) => (k - 273.15) * (9 / 5) + 32;
+
+const getLocationInfo = () => {
+	return axios
+		.get('http://localhost:5000/location', {
+			params: {
+				q: state.cityLocationEl.value,
```

Comment:
Because this pulls from the input field which starts as empty, we can't get the real time temp when we first land on the page or reset the city to Orlando. What would we need to change so that we can get the real time temp for whatever city name is being displayed?